src/content/docs/journey-app-slow/external-services.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Identify slow external services and dependencies
+metaDescription: Use the external services UI to identify slow external services in your app
+---
+
+## Objectives
+
+By the end of this tutorial, you will:
+
+* Understand how to use the external services UI
+* Have identified any slow external services such as API calls
+* Review transaction-level performance
+* Find traces that involve the transactions you're interested in
+
+## Why look at external services 
+
+The external services feature allows you to observe the upstream and downstream activity of a service. Those upstream or downstream external services may be your own services you instrumented, or more likely in this case third-party services and API calls. When these external services have a slow response time, they hold your app up causing slowness.
```

Comment:
This is the perfect length of ~conceptual content~ that a tutorial should have methinks